scripts/ci/images/Dockerfile.ci-spack-ubuntu20.04-base

```diff
@@ -26,7 +30,52 @@ RUN . /spack/share/spack/setup-env.sh && \
       --include-build-deps \
       --no-check-signature \
       --fail-fast \
-      --only dependencies && \
+      --only dependencies
+
+# # Build external mpich (using ch3:sock:tcp is faster in CI and not supported
+# # by spack), while trying to use any dependencies from the spack builds above.
+# #
+# #   configure with "--enable-g=dbg,log" for runtime debug output
+# #
+RUN . /spack/share/spack/setup-env.sh && \
+    export HWLOC_LOCATION="$(spack location --install /$(spack find --format {hash:7} hwloc))" && \
```

Comment:
Why not `spack load hwloc` before this cmd?

Reply:
I'll try that and see if it allows me to remove the specification of the `hwloc` path in the configure args.